spec/tasks/send_file_uploads_notifications_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe 'rake lumen:send_file_uploads_notifications', type: :task do
```

Comment:
I think this test is covering several cases that I would like to see separated for clarity:

1. People with `documents_notification: false` do not get email when documents update.
1. People with `documents_notification: true` _do_ get email when documents update.
1. If there are multiple people (with `documents_notification: true`) interested in the same document, they all get notified when it updates.

There is one other case that this does not cover which I would like to see tested:

1. If a document updates multiple times between task runs, the user gets only one email.

(As I understand Adam's use case, he may edit a file multiple times in quick succession, but only wants one notification email to be sent per day.)

Reply:
Ok, should be there

Reply:
All the changes to this test suite are super readable 👍